.github/ISSUE_TEMPLATE/meeting-notes.md

```diff
@@ -10,4 +10,8 @@ assignees: ''
 _Please add your topic as a comment to the issue. Use following format:_
 Topic description and link to PR, if any (duration in min)
 
+Time: 10:30am CST/CDT
```

Comment:
It's 10:00am now. The community calendar is updated. Could you please update and we can merge it. Thanks.